src/ViewHelpers/StudentApplicationFormViewHelper.php

```diff
@@ -52,9 +55,14 @@ protected static function displayPageFormTwo(array $data): string
      */
     protected static function displayPageFormThree(): string
     {
-        $output = '<div><label>Any past coding experience?</label></div>';
-        $output .= '<textarea rows="10" placeholder="Most people write a few sentences"> </textarea>';
-        $output .= '</label>';
+        $output = '<div class="row"><label for="pastCoding" class="label-control withTooltip">';
+        $output .= 'Any past coding experience?';
+        $output .= '<button class="pastCodingTooltip" data-toggle="tooltip" data-placement="top" ';
```

Comment:
Spoke to you about this - clicking this takes you back to page 1

Reply:
Removed link!